services/gateway-api/src/circuit_breaker/anthropic_breaker.py

```
@@ -0,0 +1,360 @@
+                "prompt_tokens": response.usage.prompt_tokens,
+                "completion_tokens": response.usage.completion_tokens,
+                "total_tokens": response.usage.total_tokens
+            },
+            "stop_reason": response.choices[0].finish_reason
+        }
+
+    async def _call_gemini(
+        self,
+        messages: List[Dict[str, str]],
+        system: Optional[str] = None,
+        **kwargs
```

Ruff (0.14.8)

Missing type annotation for `**kwargs`

(ANN003)